rrpproxy/rrpproxy.py

```diff
@@ -157,3 +158,13 @@ def query_foreign_transfer_list(self, **query_transfer_data):
     def query_zone_list(self):
         """Query list of activated zones in account"""
         return self.call('QueryZoneList')
+
+    def get_domain_list_from_account(self, index=0):
+        """Returns list of domains registered in your account"""
+        domain_list = self.query_domain_list(first=index)
+        next_index = domain_list['property']['last'][0] + 1
+        if domain_list['property']['total'][0] < next_index:
+            return domain_list['property']['domain'] if 'domain' in domain_list['property'] else []
+        else:
+            time.sleep(1)
```

Comment:
This is not something everyone wants per se, so I would instead give it as an optional in the function like so:
```
def get_domain_list_from_account(self, index=0, sleep_time=0):
```

Reply:
i made it customizable, but i left 1 as default value since it is probably the most common: https://github.com/ByteInternet/rrpproxy/pull/15/commits/68ad50f4484715eda1461bc435bb1362b19a2599
```
    def get_domain_list_from_account(self, index=0, time_between_calls_in_seconds=1):
        """Returns list of domains registered in your account"""
        domain_list = self.query_domain_list(first=index)
        next_index = domain_list['property']['last'][0] + 1
        if domain_list['property']['total'][0] < next_index:
            return domain_list['property']['domain'] if 'domain' in domain_list['property'] else []
        else:
            # According to RRPProxy (https://wiki.rrpproxy.net/api/epp-server/frequently-asked-questions )
            # there is a rate limit of 1 command per second, therefore I added time.sleep as a parameter so
            # we can add any custom delay between calls case we want to
            time.sleep(time_between_calls_in_seconds)
```